### ISSUE TITLE
Pipelines not working

### ISSUE THREAD

**Issue by hackforfood-uk** [alias454/graylog-bro-content-pack#2]
Hiya,

Banging my head against a brick wall a bit. I am using BRO but not in SO and trying to get the logs into graylog, thanks for all your help on the write ups etc.

Messages are in this format for the DNS log: 

1516195367.174058	CNDYE23uGhnLzFS9J8	8.8.8.8	23872	8.8.8.8	53	udp	54914	-	r4.res.office365.com	-	-	-	-	0	NOERROR	T	F	F	F	0	r4.res.office365.com.edgekey.net	300.000000	F

And using your rule file to input but for some reason getting an error:
 error=<{"type":"mapper_parsing_exception","reason":"failed to parse","caused_by":{"type":"illegal_argument_exception","reason":"Can't parse [index] value [not_analyzed] for field [id_orig_h], expected [true] or [false]"}}>

Any ideas?

**Comment by alias454** [comment]
Hi @petegriggs, The format should be pipe delimited (|) so the pipeline parsing rules on Graylog can work. I'm not 100% on differences in the fields used with a standard BRO setup vs. SO but formatting the outbound message fields with pipe separators would be a good starting point then the pipeline rules can be modified if need be.

I have only validated this against BRO instances deployed using SO but you might be able to get some hints from a section of a blog post I wrote. Some hints for a syslog_ng config are located under the 'Configure Syslog-NG on Security Onion' section of this blog post http://alias454.com/send-security-onion-logs-to-a-centralized-graylog-server/

Regards,
Brandon
